qubesappmenus/__init__.py

```diff
@@ -575,6 +584,9 @@ def appmenus_update(self, vm, force=False):
     help='required pledge for --get-available')
 parser.add_argument('domains', metavar='VMNAME', nargs='+',
     help='VMs on which perform requested actions')
+parser.add_argument('--file-field', action='append', dest='fields',
+    help='File field to append to output for --get-available; can be used'
+         ' multiple times for multiple fields')
```

Comment:
Please add info that this option change output format too. Preferably in manual page, if there would be one... But for now a note here would do.

Reply:
added a manpage